libs/parallelism/algorithms/include/hpx/parallel/algorithms/detail/is_sorted.hpp

```diff
@@ -13,15 +14,13 @@
 
 namespace hpx { namespace parallel { inline namespace v1 { namespace detail {
 
-    // provide implementation of std::distance supporting iterators/sentinels
-    // std::is_sorted is not available on all supported platforms yet
     template <typename Iter, typename Sent, typename Compare>
-    inline bool is_sorted_sequential(Iter first, Sent last, Compare const& comp)
+    inline bool is_sorted_sequential(Iter first, Sent last, Compare&& comp)
```

Comment:
I think this could be marked as `constexpr`.

Reply:
Yep, good idea!